src/main/java/sample/Controller.java

```diff
@@ -30,14 +30,14 @@ public void loginButtonIsClicked(ActionEvent event) throws IOException {
         user.setPassword(password.getText());
 
         UserVerifier userVerifier = new UserVerifier();
-        if (userVerifier.userExist(user)) {
+        if (userVerifier.userExist(user) && UserVerifier.isUserCustomerType(user)) {
```

Comment:
in UserVerifier  nu este metoda isUserCustomerType
este doar metoda  isUserEmployeeType (nu are rost sa compari daca e costomer doar daca e employee aplici discount).

Reply:
ai dreptate, lucrez pe o versiune mai veche